boards/mips/pic32mx/pic32mx-starterkit/scripts/Make.defs

```diff
@@ -49,7 +49,6 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCXXFLAGS = -fno-exceptions -fcheck-new
```

Comment:
Are all the related `defconfig` files updated to be sure that `CONFIG_CXX_EXCEPTION=y` is set explicitly?

Reply:
`CXX_EXCEPTION` default value has not been changed, so I believe it doesn't trigger any update on defconfigs.

Reply:
Yes. I missed that logic is "not equal", but was thinking that it is "equal".